cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -326,6 +336,11 @@ public void execute() {
                     "flag is not set");
         }
 
+        if (!project.buildOptions().cloud().isEmpty() && project.buildOptions().codeCoverage()) {
+            this.outStream.println("WARNING: Code coverage generation is not supported with Ballerina cloud test");
+        }
+
+
```

Comment:
```suggestion
```

Reply:
Done in [40229a4](https://github.com/ballerina-platform/ballerina-lang/pull/42183/commits/40229a40c6e8385fc2d56f004746b7a5494411f1) commit.